priv/templates/extended_bin_windows_ps

```diff
@@ -0,0 +1,311 @@
+param (
+    [string] $Command
+)
+
+# Terminate on error
+$ErrorActionPreference = "Stop"
+
+# Set variables that describe the release
+$rel_name = '{{ rel_name }}'
+$rel_vsn = '{{ rel_vsn }}'
+$erts_vsn = '{{ erts_vsn }}'
+$erl_opts = '{{ erl_opts }}'
+
+# Ensure we have PSScriptRoot
+if (!(Test-Path variable:global:PSScriptRoot)) {
+    # Support powershell 2.0
+    $PSScriptRoot = Split-Path -Parent -Path $MyInvocation.MyCommand.Definition
+}
+
+# Import psutil helper functions
+. $PSScriptRoot\psutil.ps1
+
+# Set the root release directory based on the location of this batch file
+$rootdir = Split-Path -Parent -Path $PSScriptRoot
+$rel_dir = "$rootdir\releases\$rel_vsn"
+
+$erts_root, $erts_dir = Find-ERTS -RelRoot $rootdir -Vsn $erts_vsn
+$sys_config = Find-FormatConfig -RelDir $rel_dir -File 'sys.config'
+$vm_args = Find-FormatConfig -RelDir $rel_dir -File 'vm.args'
+$boot_script = Find-BootScript -RelDir $rel_dir -RelName $rel_name
+
+$service_name = "${rel_name}_${rel_vsn}"
+$nodetool = "$rootdir\bin\nodetool"
+
+$bindir = "$erts_dir\bin"
+$werl = "$bindir\werl.exe"
+$erlsrv = "$bindir\erlsrv.exe"
+$epmd = "$bindir\epmd.exe"
+$escript = "$bindir\escript.exe"
+
+$extensions = @("{{ extensions }}" -split '[|]' | Where-Object { $_ -ne 'undefined' })
+
+# Run extension script
+function RunExtension() {
+    param(
+        [string]$Name
+    )
+    # Manifest xxx_extension as local variable
+    "{{ extension_declarations }}" -split ";" | ForEach-Object { Invoke-Expression "`$$_" }
+    # Manifest value of xxx_extension as $script
+    $script = Invoke-Expression "`$${Name}_extension"
+    # Execute
+    & "$PSScriptRoot\$script" @args
+}
+
+# Extract node type and name from vm.args
+$node_type, $node_name, $hostname = Get-NodeAndHost $vm_args
+
+# Extract the target cookie
+# Do this before relx_get_nodename so we can use it and not create a ~/.erlang.cookie
+$cookie = Get-Cookie $vm_args
+
+# Collect additional (allowed) VM args into erl_opts for erlsrv/start_erl
+$opts = Get-Content $vm_args `
+    | Select-String '^(#|-name|-sname|-setcookie|-noinput|-noshell)' -NotMatch `
+    | ForEach-Object { $_.line } `
+    | Where-Object { $_.length -gt 0 }
+$erl_opts += "$opts"
+
+# If a start.boot file is not present, copy one from the named .boot file
+if (!(Test-Path "$rel_dir\start.boot")) {
+    Copy-Item "$rel_dir\$rel_name.boot" "$rel_dir\start.boot" | Out-Null
+}
+
+# List of supported commands, help, etc
+$commands = @{
+    'install' = @{
+        'ShortHelp' = 'Install the service';
+        'LongHelp' = 'Installs release <version> as a windows service';
+        'Usage' = '<version>';
+        'Invoke' = 'Install';
+        'PassArgs' = $true
+`    };
+    'uninstall' = @{
+        'ShortHelp' = 'Uninstall the service';
+        'LongHelp' = 'Uninstalls release <version> as a windows service';
+        'Usage' = '<version>';
+        'Invoke' = 'Uninstall';
+        'PassArgs' = $true
+    };
+    'start' = @{
```

Comment:
To match the changes to the other script I think this should now be `daemon`. Unless that makes less sense on Windows?

Reply:
The equivalent of a "daemon" on windows is a "windows service":
https://en.wikipedia.org/wiki/Daemon_(computing)#Terminology

Reply:
I have added 'daemon' as an alias for 'start'.